locales/es.yaml

```diff
@@ -44,47 +44,47 @@ term:
   internalLevel: Nivel interno
   minInternalLevel: Nivel mínimo interno
   maxInternalLevel: Nivel máximo interno
-  noteDesigner: Notas del Diseñador
-  totalNotes: TOTAL
+  noteDesigner: Diseñador del mapa
+  totalNotes: Total
```

Comment:
This entry is used as the table header of "Total Note Count" in the Detail View.
The English translation is uppercased, but you can keep it capitalized if it looks more natural in Spanish.